extra_docs/ums/README.md

```diff
@@ -1,8 +1,14 @@
 # Jexia User Management System Module (UMS)
-UMS provides an ability to create user accounts, sign-in to the Jexia project with these accounts and perform 
-data changes with permissions of any authorized user. Users can be created either 
-with [Web Management Application](https://docs.jexia.com/getting-started/user-management/) or 
-with `signup()` method of this module.  
+UMS provides an ability to manage user accounts, sign-in to the Jexia project with these accounts and perform 
+data changes with permissions of any authorized user. There are a few ways to create a new users: 
+
+1. You can use [Web Management Application](https://docs.jexia.com/getting-started/user-management/) and create
+users one by one with nice UI. 
+2. You can use `signup()` method of User Management Module
+3. It is also possible to insert users as a regular data into UMS (the same way as dataset).
+
+Notice that when you create a new user any of these ways, that user will be received an activation email 
+with a link to set a password and activate his/her account. 
```

Comment:
I'd suggest changing a bit this to:
_Notice that when you create a new user by any of these ways, they will receive an activation email 
with a link to set a password and activate their account._